### ISSUE TITLE
No water and nitrogen uptakes are being supplied to the root. 

### ISSUE THREAD

**Issue by yacobbeletse** [APSIMInitiative/ApsimX#9017]
### What happened?

I am encountering some issues while testing a new crop using the Plant Modelling Framework (PMF). I am getting an error message stating: **No water and nitrogen uptakes are being supplied to the root. Is Soil Arbitrator included in the simulation?** I can confirm that the Soil Arbitrator is included in my simulation tree.

I have checked the bugs and crash reports, and while the exceptions seem similar, I have been unable to resolve the issue. The answered questions pertain to existing crops and do not address the PMF specifically.

I am unsure how to proceed further. Could you provide assistance or guidance on how to resolve this issue?
[PMFSoilArbitrator.zip](https://github.com/user-attachments/files/15717972/PMFSoilArbitrator.zip)

### What Operating System are you on?

Windows

**Comment by BrianCollinss** [comment]
Can you briefly tell us what you have changed in Root or other organs, compared to the Plant you started with?

**Comment by BrianCollinss** [comment]
Strange thing is that even when I add another plant to Paddock and sow that instead of new "Plant", I get the same error, unless I remove the "Plant". 

I am not sure if it is relevant to this issue, but you also need to add your new "Plant" to Models/Resources/SurfaceOrganicMatter.json.

**Comment by byzheng** [comment]
I worked with @yacobbeletse before he submitted this issue, but cannot figure out reason for this error. It is very strange for this simulation. 

* It should not be related with SurfaceOrganicMatter.json as I created a copy of Wheat with new name Wheat1 without any error. 
* The new `Plant` in `.Simulations.Simulation.Field.Plant` is used in the simulation even I switch to another crop in the sowing  (e.g. `Wheat`). Sure it is not used in other places and rename `Plant` to other names.
* Without any errors through disabling `.Simulations.Simulation.Field.Plant`
* Same error after enabling `.Simulations.Simulation.Field.Plant`
* It seems `Root` organ has the same property/attributes comparing with Wheat
* `DoWaterUptake` in `Root.cs` is not called in any situations https://github.com/APSIMInitiative/ApsimX/blob/6bdf11d9c69fb233d287e83169bda0ccac257c9b/Models/PMF/Organs/Root.cs#L547

Then run out of ideas for this problem.

**Comment by yacobbeletse** [comment]
@BrianCollinss -   I constructed a model called Plant in the same manner that models are coupled to build a simulation tree, that means I dragged and dropped the lower level processes to Plant

**Comment by BrianCollinss** [comment]
Have you made a change after dragging and dropping?

**Comment by yacobbeletse** [comment]
Yes, I have made changes

**Comment by peter-devoil** [comment]
Making a diff against the original resource (potato?) may uncover if a change has been made to the original that hasn't been applied to your copy. Diffs are tedious - you'll have to copy out the json to a text file and compare it against the resource file.

**Comment by byzheng** [comment]
My understanding may be not correct. A model/node in the simulation tree should not be evaluated if it is not used in other places. 

In this case @yacobbeletse's example, A new `Plant` is defined under `Paddock`, but is not used for any manager scripts. The same error still obtained.

**Comment by byzheng** [comment]
For example, the apsim file below is a minimum example which contains a new plant `Magic` and `Wheat`. Other components are removed.

* If `Magic` is disabled, no error is produced
* If `Magic` is enabled, the same error ` No water and nitrogen uptakes are being supplied to the root. Is Soil Arbitrator included in the simulation?`


[PMFSoilArbitratorMinimum.zip](https://github.com/user-attachments/files/15780930/PMFSoilArbitratorMinimum.zip)

**Comment by yacobbeletse** [comment]
@peter-devoil. In this case, I started with the Plant model instead of the Potato model. I had previously attempted to test Taro using the Potato model through Replacements. However, I moved away from using Replacements as it did not provide me the flexibility I needed to represent the specific Plant I am trying to test. For example, I could not add GenericPhase. So, the simulation with the magic Plant that Bangyou forwarded is based on the Plant model.
